### PR TITLE
GLT-2533: added Array Runs section of user manual

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: docs/user_manual/array_runs.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{% include userman-toplink.md %}
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Editing an Array Run" %}
+
+To get to the Edit Array Run page, click on the array run's ID or alias in the Array Runs list. The top section of this
+page is the same as on the Create Array Run page, and allows you to modify the array run details. Make any changes you
+would like, then click the "Save" button at the top right. The Samples section below shows all of the samples in the
+array if one has been added to the array run. Under that is another list containing the array run's change log.
```
Comment (callunity) [inline docs/user_manual/array_runs.md]: `if one has` -> `if any have`. Or, `The Samples section below shows all of the samples that have been added to the array run.`

Comment (djcooke) [reply docs/user_manual/array_runs.md]: The first suggestion implies that more than one array can be added to the run, and the second implies that samples are added directly to the array run. Reworded to "If an array has been added to the array run, all of its samples are listed in the Samples section below." which should be clearer

File: docs/user_manual/array_runs.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Creating an Array Run" %}
+
+Click the "Add" button in the toolbar at the top of the Array Runs list to get to the Create Array Run page. Here you
+can enter all of the array run details.
+
+To add an array to the run, enter the array's alias or serial number into the Array Search box and click the "Search"
+button. Matching arrays will appear in the dropdown below. Endure that the correct array is selected, then click the
```
[the remark this answers is not in the document]
Comment (djcooke) [reply docs/user_manual/array_runs.md]: fixed

File: docs/user_manual/array_runs.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+layout: page
+title: "19. Array Runs"
+section: 19
+---
+
+
+{% assign sub = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Array Runs List" %}
+
+To get to the main Array Runs list page, click "Runs" under Array Scanning in the menu on the left side of the screen.
```
[the remark this answers is not in the document]
Comment (djcooke) [reply docs/user_manual/array_runs.md]: It's capitalized when in a title, such as the Array Runs page, but not when just talking about array runs themselves

**Pull request merged**